src/middleware/FileTransferFeatureMiddleware.ts

```diff
@@ -0,0 +1,22 @@
+import { NextFunction, Request, RequestHandler, Response } from 'express';
+import { provide } from 'inversify-binding-decorators';
+import { BaseMiddleware } from 'inversify-express-utils';
+
+import { loggerInstance } from 'app/middleware/Logger';
+import { getEnv } from 'app/utils/EnvironmentUtils';
+import { ENTRY_PAGE_URI } from 'app/utils/Paths';
+
+@provide(FileTransferFeatureMiddleware)
+export class FileTransferFeatureMiddleware extends BaseMiddleware {
+
+    public handler: RequestHandler = (req: Request, res: Response, next: NextFunction): void => {
+
+        loggerInstance().info(`${FileTransferFeatureMiddleware.name} - handler: request url is ` + req.url);
```

Comment:
@n-r-b either you use string interpolation e.g. `${xxx} - ` or string concatenation e.g. `xxx + ' - '` - no point using both.

Reply:
Another thing, this log is useless.

Reply:
The reason I am saying useless is that nobody cares whether URI covered by middleware is hit. It is much useful to know that URI who is disabled was triggered.

We need to look at our logging at some point to make it more useful.